src/agents/link_creation_agent/LinkCreationRequestProxy.cc

```diff
@@ -4,17 +4,21 @@
 
 using namespace link_creation_agent;
 
-const string LinkCreationRequestProxy::Parameters::QUERY_INTERVAL =
-    "link_creation_agent.requests_interval";
-const string LinkCreationRequestProxy::Parameters::QUERY_TIMEOUT = "link_creation_agent.query_timeout";
-
-const string LinkCreationRequestProxy::Parameters::ABORT_FLAG = "link_creation_agent.abort_flag";
+const string LinkCreationRequestProxy::MAX_RESULTS = "lca.mr";
```

Comment:
Use human-readable strings here to make easier integration in the MeTTa interpreter. Something like:

```
const string LinkCreationRequestProxy::MAX_RESULTS = "MAX_RESULTS";
```

In addition to this, there's no reason to use namespaces or prefixes like `lca` because these commands are sawn/used only inside the LCA proxy.